Merkury Destination Documentation  - For both Identity & Destination destinations

framingeinstein opened the pull request.

Commit:
adding documentation

Commit:
Adding Merkury Destination documentation for pull request

Commit:
reseting

Commit:
Updateing Merkury Connections Destination

Commit:
Updating Merkury Enterprise Identity Destination

vgiurgiu commented on help/destinations/catalog/data-partners/merkury-enterprise-connections.md:
```diff
@@ -0,0 +1,245 @@
+keywords: Experience Platform;home;popular topics;destinations;connectors;destination connectors;destinations sdk;sdk;SDK
+title: Merkury Enterprise Identity Resolution Documentation self-service template for the UI
+description: Learn how to create a Merkury Enterprise Identity Resolution destionation connection using the Adobe Experience Platform UI.
+hide: true
```
not sure why we are using the `hide` and `hidefromtoc` metadata. that's only used for docs that should not be discoverable by the general public and I imagine that's not the intention. please confirm.

vgiurgiu commented on help/destinations/catalog/data-partners/merkury-enterprise-connections.md:
```diff
@@ -0,0 +1,245 @@
+The Merkury identity graph provides an average of 15% improvement in
+match rate when activating media audiences using Merkury.
+
+![A diagram of a customer data platform Description automatically
```
the styling in {} is not required

vgiurgiu commented on help/destinations/catalog/data-partners/merkury-enterprise-connections.md:
```diff
@@ -0,0 +1,245 @@
+generated](./merkury-connections/media/image1.png){width="6.5in"
+height="3.946527777777778in"}
+
+Follow these steps to create an Merkury Connections destination
```
Since the steps are not immediately after this paragraph, I'd suggest something like: 

Follow the steps in this documentation page to create a Merkury Connections destination connection and activate data using the Adobe Experience Platform user interface.

framingeinstein replied:
Addressed

vgiurgiu commented on help/destinations/catalog/data-partners/merkury-enterprise-connections.md:
```diff
@@ -0,0 +1,245 @@
+---
+# Merkury Enterprise Connections Destination
+
+Clients of Adobe Experience Platform use the Merkury Enterprise
```
Let's change from 3rd person to 2nd person please, something like: 

Use the Merkury Enterprise Connections Destination to securely deliver media audiences to Merkury. Merkury provides marketers easy matching and delivery of person-based audiences to Merkury's 80+ premium addressable TV/CTV, publisher, and ad-tech connections. Merkury is powered by a comprehensive US adult consumer identity graph of 268+ million people.

framingeinstein replied:
Addressed

vgiurgiu commented on help/destinations/catalog/data-partners/merkury-enterprise-connections.md:
```diff
@@ -0,0 +1,245 @@
+---
+# Merkury Enterprise Connections Destination
+
+Clients of Adobe Experience Platform use the Merkury Enterprise
```
Please check and correct the formatting with these line breaks in the paragraphs. Not sure where these are coming from, but they need to be removed. Thank you!

framingeinstein replied:
Addressed

vgiurgiu commented on help/destinations/catalog/data-partners/merkury-enterprise-connections.md:
```diff
@@ -0,0 +1,245 @@
+generated](./merkury-connections/media/image2.png){width="6.5in"
+height="1.7229166666666667in"}
+
+## Use Cases
```
please title the use case for easy reference

vgiurgiu commented on help/destinations/catalog/data-partners/merkury-enterprise-connections.md:
```diff
@@ -0,0 +1,245 @@
+
+## Use Cases
+
+Clients of Adobe Experience Platform use the Merkury Enterprise
```
instead of simply repeating the content from the overview, let's please add an actual description of a solved use case. See, for example, the use case from Acxiom in their destination: 

https://experienceleague.adobe.com/en/docs/experience-platform/destinations/catalog/data-partner/acxiom-data-enhancement#enhance-customer-data

framingeinstein replied:
Addressed

vgiurgiu commented on help/destinations/catalog/data-partners/merkury-enterprise-connections.md:
```diff
@@ -0,0 +1,245 @@
+
+[!IMPORTANT]
+
+-   To connect to the destination, you need the **View
```
again, not sure what happened here with the formatting, but please keep the note formatting from the documentation template that you can download from here:

https://experienceleague.adobe.com/en/docs/experience-platform/destinations/destination-sdk/document-destination/self-service-template (click on the "latest version of the template" hyperlink)

framingeinstein replied:
Addressed

vgiurgiu commented on help/destinations/catalog/data-partners/merkury-enterprise-connections.md:
```diff
@@ -0,0 +1,245 @@
+This section describes which type of audiences you can export to this
+destination.
+
+  ---------------------------------------------------------------------------------------------------------------------------------------------------------------------------------
```
similar to above, not sure what happened here, but this formatting is unsupported. please keep the formatting from the documentation template

framingeinstein replied:
Addressed

vgiurgiu commented on help/destinations/catalog/data-partners/merkury-enterprise-connections.md:
```diff
@@ -0,0 +1,245 @@
+
+  Custom uploads x               Audiences [[imported]{.underline}](https://experienceleague.adobe.com/en/docs/experience-platform/segmentation/ui/overview#import-audience) into
+                                 Experience Platform from CSV files.
+  ---------------------------------------------------------------------------------------------------------------------------------------------------------------------------------
```
we're missing the supported identities section, please see documentation template

framingeinstein replied:
Addresses

vgiurgiu commented on help/destinations/catalog/data-partners/merkury-enterprise-connections.md:
```diff
@@ -0,0 +1,245 @@
+                                 Experience Platform from CSV files.
+  ---------------------------------------------------------------------------------------------------------------------------------------------------------------------------------
+
+## Export type and frequency
```
same as further above, please keep the formatting from the documentation template

framingeinstein replied:
Addressed

vgiurgiu commented on help/destinations/catalog/data-partners/merkury-enterprise-connections.md:
```diff
@@ -0,0 +1,245 @@
+
+## Connect to the destination
+
+[!IMPORTANT]
```
incorrect formatting again for the note

framingeinstein replied:
Addressed

vgiurgiu commented on help/destinations/catalog/data-partners/merkury-enterprise-connections.md:
```diff
@@ -0,0 +1,245 @@
+To access your bucket on Experience Platform, you need to provide valid
+values for the following credentials:
+
+  ------------------------------------------------------------------------------
```
formatting, again, this is not a supported table format.

framingeinstein replied:
Addressed

vgiurgiu commented on help/destinations/catalog/data-partners/merkury-enterprise-connections.md:
```diff
@@ -0,0 +1,245 @@
+
+### New Account
+
+To define a new Merkury Managed S3 location:
```
there is a screenshot here but no instructions at all. we could stick to the format and the description in the Authenticate to destination section of the self-service template

framingeinstein replied:
Addressed

vgiurgiu commented on help/destinations/catalog/data-partners/merkury-enterprise-connections.md:
```diff
@@ -0,0 +1,245 @@
+generated](./merkury-connections/media/image5.png){width="7.260470253718285in"
+height="1.5785312773403324in"}
+
+### **Fill in destination details
```
looks like a stray ** here

framingeinstein replied:
Addressed

vgiurgiu commented on help/destinations/catalog/data-partners/merkury-enterprise-connections.md:
```diff
@@ -0,0 +1,245 @@
+
+### **Fill in destination details
+
+To configure details for the destination, fill in the required and
```
formatting again, not sure why we have the line breaks but please remove them

framingeinstein replied:
Addressed

vgiurgiu commented on help/destinations/catalog/data-partners/merkury-enterprise-connections.md:
```diff
@@ -0,0 +1,245 @@
+optional fields below. An asterisk next to a field in the UI indicates
+that the field is required.
+
+![A screenshot of a computer Description automatically
```
the alt-text is not useful, it should describe what's happening in the screenshot.

framingeinstein replied:
Addressed

vgiurgiu commented on help/destinations/catalog/data-partners/merkury-enterprise-connections.md:
```diff
@@ -0,0 +1,245 @@
+
+Follow these steps to create an Merkury Connections destination
+connection and dataflow using the Adobe Experience Platform user
+interface.![A screenshot of a computer Description automatically
```
the alt-text is not useful, it should describe what's happening in the screenshot.

framingeinstein replied:
Addressed

vgiurgiu commented on help/destinations/catalog/data-partners/merkury-enterprise-connections.md:
```diff
@@ -0,0 +1,245 @@
+generated](./merkury-connections/media/image6.png){width="6.5in"
+height="3.877083333333333in"}
+
+-   **Name (Required)** - The name the destination will be saved under
```
formatting, please keep the formatting from the template

framingeinstein replied:
Addressed

vgiurgiu commented on help/destinations/catalog/data-partners/merkury-enterprise-connections.md:
```diff
@@ -0,0 +1,245 @@
+    the exported files. Currently, the only file type Acxiom processing
+    will be expecting is CSV
+
+![A white background with black text Description automatically
```
we shouldn't use images as notes

framingeinstein replied:
Addressed

vgiurgiu commented on help/destinations/catalog/data-partners/merkury-enterprise-connections.md:
```diff
@@ -0,0 +1,245 @@
+generated](./merkury-connections/media/image7.png){width="6.5in"
+height="1.336111111111111in"}
+
+![A screenshot of a computer Description automatically
```
same as further above about the alt-text

framingeinstein replied:
Addressed

vgiurgiu commented on help/destinations/catalog/data-partners/merkury-enterprise-connections.md:
```diff
@@ -0,0 +1,245 @@
+destinations](https://experienceleague.adobe.com/en/docs/experience-platform/destinations/ui/activate/activate-batch-profile-destinations) for
+instructions on activating audiences to this destination.
+
+## Mapping suggestions
```
I noticed this section is copied from the Acxiom page, do the same mapping suggestions apply to the Merkury destination? 

https://experienceleague.adobe.com/en/docs/experience-platform/destinations/catalog/data-partner/acxiom-data-enhancement#mapping-suggestions

framingeinstein replied:
Addressed

vgiurgiu commented on help/destinations/catalog/data-partners/merkury-enterprise-connections.md:
```diff
@@ -0,0 +1,245 @@
+
+## Next steps
+
+By following this tutorial, you have successfully created a dataflow to
```
Again, I notice that this is copied from https://experienceleague.adobe.com/en/docs/experience-platform/destinations/catalog/data-partner/acxiom-data-enhancement#exported-data . Is the process the same for  the Merkury destination?

framingeinstein replied:
Addressed

vgiurgiu reviewed (changes requested):
hi @framingeinstein - thank you for the documentation updates! There are numerous comments that we need to address before we can publish. Some of the most important things: 

* Looks like paragraph formatting is off throughout the doc. Not sure why this is, it may be from your editor? In any case, please correct and resubmit
* I would suggest if possible to keep the two destinations separated in two separate PRs. For example, if by any chance the destinations are set live at different times, we want to keep the two documentation pages independent of each other
* I notice that you added the images to the same folder structure as the .md files. Please follow the convention in step four of this section: https://experienceleague.adobe.com/en/docs/experience-platform/destinations/destination-sdk/document-destination/work-in-local-environment#author-documentation . 
* I notice that a lot of the content is copied from the Acxiom destination pages (e.g. mapping suggestions, validate data export section). That's fine as long as the behavior of your destination is the same or as long as the process to validate the data export is the same. If they are not, please make the required edits to best describe the behavior and processes for your destination.

I'm happy to connect over a call if any of the requirements are unclear.

Commit:
updates have been made per pull request comments.

framingeinstein commented:
I have made updates per the comments and committed the changes.  

Over all:
Fixed:Looks like paragraph formatting is off throughout the doc. Not sure why this is, it may be from your editor? In any case, please correct and resubmit
Not Fixed: I would suggest if possible to keep the two destinations separated in two separate PRs. For example, if by any chance the destinations are set live at different times, we want to keep the two documentation pages independent of each other
Fixed: I notice that you added the images to the same folder structure as the .md files. Please follow the convention in step four of this section: https://experienceleague.adobe.com/en/docs/experience-platform/destinations/destination-sdk/document-destination/work-in-local-environment#author-documentation .
Fixed: I notice that a lot of the content is copied from the Acxiom destination pages (e.g. mapping suggestions, validate data export section). That's fine as long as the behavior of your destination is the same or as long as the process to validate the data export is the same. If they are not, please make the required edits to best describe the behavior and processes for your destination.

At this point we would prefer to keep the pull request as one for both destinations.

vgiurgiu commented on help/destinations/catalog/data-partners/yourdestination-template 3.md:
```diff
@@ -0,0 +1,200 @@
+---
```
I notice you added this template by mistake to the PR. Just a reminder to remove it please.

framingeinstein replied:
Removed

framingeinstein replied:
Removed

vgiurgiu commented on help/destinations/catalog/data-partners/merkury-enterprise-connections.md:
```diff
@@ -0,0 +1,160 @@
+keywords: Experience Platform;home;popular topics;destinations;connectors;destination connectors;destinations sdk;sdk;SDK
+title: Merkury Enterprise Identity Resolution Documentation self-service template for the UI
+description: Learn how to create a Merkury Enterprise Identity Resolution destionation connection using the Adobe Experience Platform UI.
+hide: false
```
same comment as before about the hide and hidefromtoc metadata. this will make the page not be discoverable in our catalog and table of contents and is meant for limited releases that we only want to share with select customers. is that your intention? if not, please remove these two lines from the metadata.

framingeinstein replied:
Removed

framingeinstein replied:
Removed

vgiurgiu commented on help/destinations/catalog/data-partners/merkury-enterprise-connections.md:
```diff
@@ -0,0 +1,160 @@
+---
+# Merkury Enterprise Connections Destination
+
+## Overview
```
nitpicky, but please leave a blank line between the heading and the paragraph.

framingeinstein replied:
Addressed

framingeinstein replied:
Addressed

vgiurgiu commented on help/destinations/catalog/data-partners/merkury-enterprise-connections.md:
```diff
@@ -0,0 +1,160 @@
+## Overview
+Use the Merkury Enterprise Identity Destination to securely deliver media audiences to Merkury. Merkury provides marketers easy matching and delivery of person-based audiences to Merkury's 80+ premium addressable TV/CTV, publisher, and ad-tech connections. Merkury is powered by a comprehensive US adult consumer identity graph of 268+ million people.
+
+![A diagram of a customer data platform Description](../../assets/catalog/data-partners/merkury-connections/media/image1.png)
```
alt-text should be more descriptive, something like:

"a diagram showing the interconnection between Merkury and Experience Platform, including ingestion and activation"

framingeinstein replied:
Addressed

vgiurgiu commented on help/destinations/catalog/data-partners/merkury-enterprise-connections.md:
```diff
@@ -0,0 +1,160 @@
+
+![A diagram of a customer data platform Description](../../assets/catalog/data-partners/merkury-connections/media/image1.png)
+
+Follow the steps in this documentation page to create a Merkury Identity destination connection and activate data using the Adobe Experience Platform user interface.![A screenshot of a computer Description](../../assets/catalog/data-partners/merkury-connections/media/image2.png)
```
alt-text should be more descriptive, something like:

"The Merkury Enterprise Conections destination card highlighted in the Experience Platform destinations catalog."

framingeinstein replied:
Addressed

framingeinstein replied:
Addressed

vgiurgiu commented on help/destinations/catalog/data-partners/merkury-enterprise-connections.md:
```diff
@@ -0,0 +1,160 @@
+
+## Prerequisites
+
+[!IMPORTANT]
```
the note is still not properly formatted, plase check formatting in the template. it should be something like: 

>[!IMPORTANT]
>
>Content
>
>More content if desired

vgiurgiu commented on help/destinations/catalog/data-partners/merkury-enterprise-connections.md:
```diff
@@ -0,0 +1,160 @@
+![A diagram of a customer data platform Description](../../assets/catalog/data-partners/merkury-connections/media/image1.png)
+
+Follow the steps in this documentation page to create a Merkury Identity destination connection and activate data using the Adobe Experience Platform user interface.![A screenshot of a computer Description](../../assets/catalog/data-partners/merkury-connections/media/image2.png)
+
```
Please add a disclaimer note about this being a partner-authored connector and provide users with a means to contact your team. See the example in the documentation template.

```
>[!IMPORTANT]
>
>The destination connector and documentation page are created and maintained by the *YourDestination* team. For any inquiries or update requests, please contact them directly at *Insert link or email address where you can be reached for updates, for example `support@YourDestination.com`.*
```

vgiurgiu replied:
where is the disclaimer? please add it. @framingeinstein

vgiurgiu commented on help/destinations/catalog/data-partners/merkury-enterprise-connections.md:
```diff
@@ -0,0 +1,160 @@
+
+|Target Identity|Description|Considerations|
+|---|---|---|
+|GAID|Google Advertising ID|Select the GAID target identity when your source identity is a GAID namespace.|
```
just checking - will exporting all these identities be supported by your destination? please check against the configuration you submitted to us and confirm.

vgiurgiu commented on help/destinations/catalog/data-partners/merkury-enterprise-connections.md:
```diff
@@ -0,0 +1,160 @@
+## Connect to the destination
+
+[!IMPORTANT]
+To connect to the destination, you need the **View Destinations** and **Manage and Activate Dataset Destinations** [[access control permissions]{.underline}](https://experienceleague.adobe.com/en/docs/experience-platform/access-control/home#permissions). Read the [[access control overview]{.underline}](https://experienceleague.adobe.com/en/docs/experience-platform/access-control/ui/overview) or c ...
```
the formatting for the note is still off. Please see the examples further above and in the documentation template of how the note should be formatted.

vgiurgiu commented on help/destinations/catalog/data-partners/merkury-enterprise-connections.md:
```diff
@@ -0,0 +1,160 @@
+
+To connect to this destination, follow the steps described in the [[destination configuration tutorial]{.underline}](https://experienceleague.adobe.com/en/docs/experience-platform/destinations/ui/connect-destination). In the destination configuration workflow, fill in the fields listed in the two sections below.
+
+### New Account
```
we can drop this heading

vgiurgiu replied:
@framingeinstein please remember to address this comment

vgiurgiu commented on help/destinations/catalog/data-partners/merkury-enterprise-connections.md:
```diff
@@ -0,0 +1,160 @@
+
+### New Account
+
+#### Authenticate to destination
```
please promote this to a ### heading after removing the heading above, same for Fill in destination details further below

vgiurgiu replied:
@framingeinstein please remember to address this comment

vgiurgiu commented on help/destinations/catalog/data-partners/merkury-enterprise-connections.md:
```diff
@@ -0,0 +1,160 @@
+
+{style="table-layout:auto"}
+
+![A screenshot of a computer Description automatically generated](../../assets/catalog/data-partners/merkury-connections/media/image4.png)
```
again, the alt-text is not helpful. based on the examples provided further above, please update the alt-text

vgiurgiu replied:
@framingeinstein please remember to address this comment

vgiurgiu commented on help/destinations/catalog/data-partners/merkury-enterprise-connections.md:
```diff
@@ -0,0 +1,160 @@
+
+To configure details for the destination, fill in the required and optional fields below. An asterisk next to a field in the UI indicates that the field is required.
+
+![A screenshot of a computer Description automatically generated](../../assets/catalog/data-partners/merkury-connections/media/image6.png)
```
same comment about alt-text

vgiurgiu commented on help/destinations/catalog/data-partners/merkury-enterprise-connections.md:
```diff
@@ -0,0 +1,160 @@
+![A screenshot of a computer Description automatically generated](../../assets/catalog/data-partners/merkury-connections/media/image6.png)
+
+-   **Name (Required)** - The name the destination will be saved under
+-   **Description** - Short explanation of the destination's purpose
```
can we use * instead of - for unordered lists please, same as in the template?

vgiurgiu replied:
@framingeinstein please remember to address this comment

vgiurgiu commented on help/destinations/catalog/data-partners/merkury-enterprise-connections.md:
```diff
@@ -0,0 +1,160 @@
+-   **Description** - Short explanation of the destination's purpose
+-   **Bucket Name (Required)** - Name of the Amazon S3 bucket set up on S3
+-   **Folder Path (Required)** - If subdirectories in a bucket are used a path must be defined, or '/' to reference the root path.
+-   **File Type** - Select the format Experience Platform should use for the exported files. Currently, the only file type Acxiom processing will be expecting is CSV
```
please update Acxiom to the name of your destination

vgiurgiu commented on help/destinations/catalog/data-partners/merkury-enterprise-connections.md:
```diff
@@ -0,0 +1,160 @@
+-   **File Type** - Select the format Experience Platform should use for the exported files. Currently, the only file type Acxiom processing will be expecting is CSV
+
+[!NOTE]
+When selecting the CSV option, Delimiter, Quote Character, Escape Character, Empty Value, Null Value, Compression format, and Include manifest file options will be presented, the following document explains these settings in more detail configure the formatting options.
```
there is a mention of "the following document" that explains in more detail the formatting options, but no link to the doc itself. Again, if copying material from the Acxiom destination, please make sure we do it properly and maintain useful links etc.

framingeinstein replied:
Fixed

framingeinstein replied:
Addressed

vgiurgiu commented on help/destinations/catalog/data-partners/merkury-enterprise-connections.md:
```diff
@@ -0,0 +1,160 @@
+
+## Activate audiences to this destination
+
+[!IMPORTANT]
```
note formatting again

framingeinstein replied:
Addressed

vgiurgiu commented on help/destinations/catalog/data-partners/merkury-enterprise-connections.md:
```diff
@@ -0,0 +1,160 @@
+[!IMPORTANT]
+- To activate data, you need the View Destinations, Activate Destinations, View Profiles, and View Segments access control permissions. Read the access control overview or contact your product administrator to obtain the required permissions.
+- To export identities, you need the View Identity Graph access control permission.
+
```
unnecessary blank line

framingeinstein replied:
Addressed, I think

vgiurgiu commented on help/destinations/catalog/data-partners/merkury-enterprise-connections.md:
```diff
@@ -0,0 +1,160 @@
+
+|Target Field|Source Description|
+|---|---|
+|id|Identity field to be used to map merkury data to AEP through the Merkury Enterprise Identity Resolution Source connector|
```
please use Experience Platform instead of AEP as that is not a public-facing acronym

framingeinstein replied:
Addressed

vgiurgiu commented on help/destinations/catalog/data-partners/merkury-enterprise-connections.md:
```diff
@@ -0,0 +1,160 @@
+|Target Field|Source Description|
+|---|---|
+|id|Identity field to be used to map merkury data to AEP through the Merkury Enterprise Identity Resolution Source connector|
+|Input_First_Name|The person. name. firstName value in Experience Platform.|
```
please no spaces in `person. name. firstName`. suggestion also to wrap these attributes in `` so they stand out on the page

framingeinstein replied:
Addressed

vgiurgiu reviewed (changes requested):
Still lots of comments to address, particularly around formatting. While for time-saving purposes and to avoid redundancy, I left the comments on the Enterprise Connections destination only, please be sure to review and address the comments for the Identity destination too, since a vast majority of them apply for both destinations. 

Thank you!

Commit:
Corrections per pull request comments.

framingeinstein commented:
I have made the updates.  Let me know if we missed anything.

vgiurgiu commented on help/destinations/catalog/data-partners/merkury-enterprise-connections.md:
```diff
@@ -0,0 +1,158 @@
+
+## Overview
+
+Use the Merkury Enterprise Identity Destination to securely deliver media audiences to Merkury. Merkury provides marketers easy matching and delivery of person-based audiences to Merkury's 80+ premium addressable TV/CTV, publisher, and ad-tech connections. Merkury is powered by a comprehensive US adult consumer identity graph of 268+ million people.
```
nitpicky comment but why does it need to be "media" audiences and not simply audiences? 

also, in the overview part, it would be a nice-to-have to clarify how the use case for this destination is different to the other destination that you are building. basically, I'm a reader and prospective user, can you briefly and succinctly tell me when to use this destination and when to use the other one?

framingeinstein replied:
I have updated this to remove the "media" qualifier from audiences.  I have also added a note to indicate when they should be using the other destination for both destinations.

vgiurgiu commented on help/destinations/catalog/data-partners/merkury-enterprise-connections.md:
```diff
@@ -0,0 +1,158 @@
+
+## Overview
+
+Use the Merkury Enterprise Identity Destination to securely deliver media audiences to Merkury. Merkury provides marketers easy matching and delivery of person-based audiences to Merkury's 80+ premium addressable TV/CTV, publisher, and ad-tech connections. Merkury is powered by a comprehensive US adult consumer identity graph of 268+ million people.
```
this doc is about the enterprise connections destination, yet we mention identity destination at the beginning of the overview. please correct.

framingeinstein replied:
Corrected.

vgiurgiu commented on help/destinations/catalog/data-partners/merkury-enterprise-connections.md:
```diff
@@ -0,0 +1,158 @@
+
+## Prerequisites
+
+>[!IMPORTANT]
```
almost there, but the table formatting is still not correct :) 

it should be 

```
>[!IMPORTANT]
>
>content

```

and please use * for bullet points if possible

framingeinstein replied:
Addressed

vgiurgiu commented on help/destinations/catalog/data-partners/merkury-enterprise-connections.md:
```diff
@@ -0,0 +1,158 @@
+
+To connect to this destination, follow the steps described in the [[destination configuration tutorial]{.underline}](https://experienceleague.adobe.com/en/docs/experience-platform/destinations/ui/connect-destination). In the destination configuration workflow, fill in the fields listed in the two sections below.
+
+### New Account
```
please remember to address this comment

framingeinstein replied:
What is this comment referring to?  I am removing this block as it seems out of place.  Let me know if you disagree.

vgiurgiu replied:
agreed, it's about removing ### New Account and updating the headings just below from `####` to `###`

framingeinstein replied:
Fixed

vgiurgiu commented on help/destinations/catalog/data-partners/merkury-enterprise-identity.md:
```diff
@@ -0,0 +1,168 @@
+
+Accounts already defined using the Merkury Enterprise Identity destination appear in a list pop-up. When selected, you can see details on the account in the right rail. View the example from the UI, when you navigate to **Destinations** \> **Accounts**;
+
+![A screenshot of destionation account in detaination accounts page](../../assets/catalog/data-partners/merkury-identity/media/image5.png)
```
minor, but watch the typos please:

destionation
detaination

framingeinstein replied:
Fixed in both files.

vgiurgiu commented on help/destinations/catalog/data-partners/merkury-enterprise-identity.md:
```diff
@@ -0,0 +1,168 @@
+![A screenshot of destination details](../../assets/catalog/data-partners/merkury-identity/media/image6.png)
+
+
+-   **Name (Required)** - The name the destination will be saved under
```
can we please use * instead of - in the list for consistency with the template and other docs?

framingeinstein replied:
I have updated all bullets from - to * for all lists

vgiurgiu replied:
much appreciated, thank you!

vgiurgiu commented on help/destinations/catalog/data-partners/merkury-enterprise-connections.md:
```diff
@@ -0,0 +1,158 @@
+---
+keywords: Experience Platform;home;popular topics;destinations;connectors;destination connectors;destinations sdk;sdk;SDK
+title: Merkury Enterprise Identity Resolution Documentation self-service template for the UI
```
check the title: Merkury Enterprise Connections Destination instead of what you have there now

framingeinstein replied:
Fixed

vgiurgiu commented on help/destinations/catalog/data-partners/merkury-enterprise-connections.md:
```diff
@@ -0,0 +1,158 @@
+---
+keywords: Experience Platform;home;popular topics;destinations;connectors;destination connectors;destinations sdk;sdk;SDK
+title: Merkury Enterprise Identity Resolution Documentation self-service template for the UI
+description: Learn how to create a Merkury Enterprise Identity Resolution destionation connection using the Adobe Experience Platform UI.
```
enterprise connections in this doc, not identity

framingeinstein replied:
Fixed

vgiurgiu commented on help/destinations/catalog/data-partners/merkury-enterprise-identity.md:
```diff
@@ -0,0 +1,168 @@
+
+## Overview
+
+Clients of Adobe Experience Platform use the Merkury Enterprise Identity Destination to build more accurate, comprehensive, and insightful consumer profiles. With improved profile data, Marketers can power better insights, segments, and models resulting in more accurate targeting and predictive modeling.
```
same as for the other destination, rephrase please to use 2nd person instead of 3rd person. Start with "Use the Merkury Enterprise Identity Destination to build more accurate [...]"

Marketers should be lowercase in the middle of a sentence

framingeinstein replied:
Fixed

vgiurgiu commented on help/destinations/catalog/data-partners/merkury-enterprise-identity.md:
```diff
@@ -0,0 +1,168 @@
+
+Clients of Adobe Experience Platform use the Merkury Enterprise Identity Destination to build more accurate, comprehensive, and insightful consumer profiles. With improved profile data, Marketers can power better insights, segments, and models resulting in more accurate targeting and predictive modeling.
+
+![A diagram of a customer data platform Description automatically
```
alt-text not helpful

framingeinstein replied:
Fixed

vgiurgiu commented on help/destinations/catalog/data-partners/merkury-enterprise-identity.md:
```diff
@@ -0,0 +1,168 @@
+---
+keywords: Experience Platform;home;popular topics;destinations;connectors;destination connectors;destinations sdk;sdk;SDK
+title: Merkury Enterprise Connections Documentation self-service template for the UI
```
this doc is about identity, update title here please

framingeinstein replied:
Fixed

vgiurgiu commented on help/destinations/catalog/data-partners/merkury-enterprise-identity.md:
```diff
@@ -0,0 +1,168 @@
+![A diagram of a customer data platform Description automatically
+generated](../../assets/catalog/data-partners/merkury-identity/media/image1.png)
+
+Follow the steps in this documentation page to create a Merkury Connections destination connection and activate data using the Adobe Experience Platform user interface.
```
this doc is about identity, update description here please

framingeinstein replied:
Fixed

vgiurgiu commented on help/destinations/catalog/data-partners/merkury-enterprise-identity.md:
```diff
@@ -0,0 +1,168 @@
+-   **Identity Resolution**: Build an accurate and comprehensive single view of the customer, informed by Merkury Individual and Household IDs. Merkury IDs provide a deep level of profile linking powered by Merkury's comprehensive US adult consumer identity graph of 268+ million people.
+-   **Enrichment**: Drive better insights and personalization with Merkury Data. Merkury Data includes over 10,000+ available data attributes ranging from demographic, lifestyle, financial, life events, and purchase data from the Merkury Data Suite.
+
+[!NOTE]
```
note formatting incorrect. please check doc template and see my other comments for correct formatting.

framingeinstein replied:
I did not see an example of the note in the template.  I have updated based on other comments.

vgiurgiu replied:
[yourdestination-template.md](https://github.com/user-attachments/files/15518175/yourdestination-template.md)

Hi @framingeinstein, see the attached file and go to lines 16, 95, or 131 to see examples of properly formatted notes. 
You can see how all this renders in 
https://experienceleague.adobe.com/en/docs/experience-platform/destinations/destination-sdk/document-destination/self-service-template

Let me know if this makes sense now.

vgiurgiu commented on help/destinations/catalog/data-partners/merkury-enterprise-identity.md:
```diff
@@ -0,0 +1,168 @@
+
+To connect to this destination, follow the steps described in the [[destination configuration tutorial]{.underline}](https://experienceleague.adobe.com/en/docs/experience-platform/destinations/ui/connect-destination). In the destination configuration workflow, fill in the fields listed in the two sections below.
+
+### New Account
```
same as in the other destination, you can remove this heading. and please update Authenticate to destination and Fill in destination details to `###` headings

framingeinstein replied:
Fixed

vgiurgiu commented on help/destinations/catalog/data-partners/merkury-enterprise-identity.md:
```diff
@@ -0,0 +1,168 @@
+
+{style="table-layout:auto"}
+
+![A screenshot of a computer Description automatically generated](../../assets/catalog/data-partners/merkury-identity/media/image4.png)
```
alt-text not helpful

framingeinstein replied:
Fixed

vgiurgiu commented on help/destinations/catalog/data-partners/merkury-enterprise-identity.md:
```diff
@@ -0,0 +1,168 @@
+---
+
+# Merkury Enterprise Identity Destination
+
```
need a top-of-page note about this being a partner-authored and managed destination. see my comment on the other destination and the doc template for an example.

framingeinstein replied:
Done

vgiurgiu commented on help/destinations/catalog/data-partners/merkury-enterprise-identity.md:
```diff
@@ -0,0 +1,168 @@
+Follow the steps in this documentation page to create a Merkury Connections destination connection and activate data using the Adobe Experience Platform user interface.
+
+![A diagram showing the interconnection between Merkury and Experience Platform, including ingestion and activation](../../assets/catalog/data-partners/merkury-identity/media/image2.png)
+
```
can we also add a brief note or paragraph about how this destination is different to the other Merkury destination and when to use each of them?

framingeinstein replied:
I have updated this to have a note about the other connector.

vgiurgiu commented on help/destinations/catalog/data-partners/merkury-enterprise-identity.md:
```diff
@@ -0,0 +1,168 @@
+>[!IMPORTANT]
+>To connect to the destination, you need the **View Destinations** and **Manage and Activate Dataset Destinations** [[access control permissions]{.underline}](https://experienceleague.adobe.com/en/docs/experience-platform/access-control/home#permissions). Read the [[access control overview]{.underline}](https://experienceleague.adobe.com/en/docs/experience-platform/access-control/ui/overview) or  ...
+
+To connect to this destination, follow the steps described in the [[destination configuration tutorial]{.underline}](https://experienceleague.adobe.com/en/docs/experience-platform/destinations/ui/connect-destination). In the destination configuration workflow, fill in the fields listed in the two sections below.
```
this formatting for linking is incorrect, please follow the one from the doc template

vgiurgiu commented on help/destinations/catalog/data-partners/merkury-enterprise-identity.md:
```diff
@@ -0,0 +1,168 @@
+
+## Connect to the destination
+
+>[!IMPORTANT]
```
incorrect note formatting, please update

framingeinstein replied:
@vgiurgiu, the call out here is for a [!IMPORTANT], I have updated the one [!NOTE] item to match the [!IMPORTANT] changes.  Let me know if this is what you were referring to or not.

Like so:
```
>[!NOTE]

>When selecting the CSV option....
```

vgiurgiu replied:
hey @framingeinstein , it's fine to leave it as IMPORTANT, my formatting comment relates to the way you use the chevrons. The formatting should be like below (notice that I have a chevron where you have a blank line):

```
>[!IMPORTANT]
>
>When selecting the CSV option....
```

vgiurgiu commented on help/destinations/catalog/data-partners/merkury-enterprise-identity.md:
```diff
@@ -0,0 +1,168 @@
+
+## Activate audiences to this destination
+
+>[!IMPORTANT]
```
incorrect note formatting, please update

framingeinstein replied:
@vgiurgiu, the call out here is for a [!IMPORTANT], I have updated the one [!NOTE] item to match the [!IMPORTANT] changes.  Let me know if this is what you were referring to or not.

Like so:
```
>[!NOTE]

>When selecting the CSV option....
```

vgiurgiu commented on help/destinations/catalog/data-partners/merkury-enterprise-identity.md:
```diff
@@ -0,0 +1,168 @@
+
+By following this tutorial, you have successfully created a dataflow to export profile data from Experience Platform to your Merkury managed S3 location. Next, you need to contact your Merkury representative with the name of the account, file names, and the bucket path so that processing can set up.
+
+## Data usage and governance
```
minor, but can we reverse this section and the Next steps section further above, so the article ends on the Next steps please?

framingeinstein replied:
Addressed

vgiurgiu reviewed (commented):
Plenty of comments still to address. Please make sure you read and address all my comments. We can also connect over a call if you have questions. Thank you!

Commit:
Updated per comments

Commit:
Updated per comments

framingeinstein commented:
I have gone through all the comments and made the updates.  If I missed anything, we can discuss on the call Monday

vgiurgiu commented:
> I have gone through all the comments and made the updates. If I missed anything, we can discuss on the call Monday

@framingeinstein sounds good & thanks. I see that we have a call scheduled for Wednesday. I'll make sure to give this another read by then. One more quick ask - can you update the name of the PR to account for the fact that it actually includes the two destinations that your team is authoring?

Commit:
removing underlines and note formating

vgiurgiu commented on help/destinations/catalog/data-partners/merkury-enterprise-connections.md:
```diff
@@ -0,0 +1,169 @@
+---
+keywords: Experience Platform;home;popular topics;destinations;connectors;destination connectors;destinations sdk;sdk;SDK
+title: Merkury Enterprise Connections Documentation self-service template for the UI
```
in the title metadata, remove the part "self-service template for the UI"

vgiurgiu commented on help/destinations/catalog/data-partners/merkury-enterprise-connections.md:
```diff
@@ -0,0 +1,169 @@
+---
+keywords: Experience Platform;home;popular topics;destinations;connectors;destination connectors;destinations sdk;sdk;SDK
+title: Merkury Enterprise Connections Documentation self-service template for the UI
+description: Learn how to create a Merkury Enterprise Connections destionation connection using the Adobe Experience Platform UI.
```
typo: destionation

vgiurgiu commented on help/destinations/catalog/data-partners/merkury-enterprise-connections.md:
```diff
@@ -0,0 +1,169 @@
+
+>[!NOTE]
+>
+>If you are looking to activate audiences to media destinations with you Merkury Connect account, use our Merkury Connections destination instead.
```
this note is excellent, but it belongs to the Identity document! one additional comment - should be _your_ Merkury account

vgiurgiu commented on help/destinations/catalog/data-partners/merkury-enterprise-identity.md:
```diff
@@ -0,0 +1,178 @@
+---
+keywords: Experience Platform;home;popular topics;destinations;connectors;destination connectors;destinations sdk;sdk;SDK
+title: Merkury Enterprise Identity Documentation self-service template for the UI
```
same comment as for the other page, please remove "self-service template for the UI" from the title metadata

vgiurgiu reviewed (commented):
just some minor comments left. but please also discuss with the Merkury team the question of supported identities for export, as discussed in our meeting yesterday
